helm_deploy/laa-estimate-eligibility/templates/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "app.fullname" . }}
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "app.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "app.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: api
+          image: '{{ .Values.image.repository }}:{{ .Values.image.tag }}'
+          imagePullPolicy: IfNotPresent
+  {{ include "app.envs" . | nindent 10 }}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /estimates/1/build_estimates/new
+              port: http
+            initialDelaySeconds: 30
+          readinessProbe:
+            httpGet:
+              path: /estimates/1/build_estimates/new
+              port: http
+            initialDelaySeconds: 30
```

Comment:
This is going to break with the next PR merge. Probably better to hit the root path and make sure that the root is mapped ?

Reply:
I think we need another ticket for this. Essentially I feel this ticket is big enough as it currently is. Looking at CFE/Apply they have implemented a ping page for the K8s readiness/liveness probe. Its probably worth discussing what approach we want to take for EFE. Unless the next PR merge removes that route it should work fine until we decide on a more permanent solution.

Reply:
Can we throw the most basicest of routes in to get a bit of a best of both? I got the impression from @starswan's comment that that's exactly what the next PR will do.
If we went with /ping/ just returning the most absolute bare mininum 200 response for now that'd not be too much work I think and would mean other bits of work don't have to worry aobut compaitbility for now (which they shouldn't have to), without committing us to a solution for what the real probe endpoints will look like

Reply:
But yes agree we should have a ticket for deciding on and creating the live and ready endpoints

Reply:
basic status page added and readiness/liveness endpoint updated